master/usage/routereflector/calico-routereflector.md

```diff
@@ -0,0 +1,294 @@
+---
+title: 'Calico BIRD Route Reflector container'
```

Comment:
i'm on the fence - i'm erring on usage though because it isn't really a reference doc about the container image, it is about how to spin it up - more tutorial style.